magprime/automated_emails.py

```diff
@@ -57,27 +57,24 @@ def __init__(self, event):
 StopsEmail('MAGFest Staff Support', 'staff_support.txt',
            lambda a: a.assigned_to(c.STAFF_SUPPORT) and not a.trusted_in(c.STAFF_SUPPORT))
 
-MarketplaceEmail('Your MAGFest Marketplace Application', 'marketplace_delay.txt',
-                 lambda g: g.status == c.UNAPPROVED and g.registered < datetime(2015, 10, 29, tzinfo=c.EVENT_TIMEZONE))
-
 # Left here in case the decision to remove is re-visited later. 20160809 RAE
 # MarketplaceEmail('Your MAGFest Marketplace Application has been waitlisted', 'marketplace_auto_waitlisted.txt',
 #                 lambda g: g.status == c.WAITLISTED and g.registered >= c.DEALER_REG_DEADLINE)
 
 StopsEmail('MAGFest Dept Checklist Introduction', 'dept_checklist_intro.txt',
            lambda a: a.is_single_dept_head and a.admin_account)
 
-AutomatedEmail(Attendee, 'Last Chance for MAGFest 2016 bonus swag!', 'attendee_swag_promo.html',
+AutomatedEmail(Attendee, 'Last Chance for MAGFest ' c.YEAR ' bonus swag!', 'attendee_swag_promo.html',
                lambda a: a.can_spam and
```

Comment:
alternatively, if we change https://github.com/magfest/ubersystem/blob/master/uber/automated_emails.py#L19

from
```self.subject = subject.format(EVENT_NAME=c.EVENT_NAME)```

to
```self.subject = subject.format(EVENT_NAME=c.EVENT_NAME, YEAR=c.YEAR)```

then you can write it like this:

```'Last Chance for {EVENT_NAME} {YEAR} bonus swag!'```